polaris/polaris/management/commands/process_pending_deposits.py

```diff
@@ -192,23 +199,207 @@ async def process_deposit(cls, transaction, server, locks):
+                )
+                if await cls.is_account_ready(transaction, server, locks):
+                    qa.queue_transaction(
+                        "check_accounts_task", SUBMIT_TRX_QUEUE, transaction
+                    )
+
+    @classmethod
+    async def is_account_ready(
+        cls, transaction: Transaction, server: ServerAsync, locks: Dict
+    ):
+        logger.info(f"check_accounts_task - processing transaction {transaction.id}")
         if await cls.requires_trustline(transaction, server, locks):
-            return
+            logger.info(
+                f"transaction {transaction.id} requires a trustline, continuing with "
+                "next transaction..."
+            )
+            return False
+
         logger.info(f"transaction {transaction.id} has the appropriate trustline")
-        await cls.handle_submit(transaction, server, locks)
+
+        logger.info(
+            f"check_accounts_task - saving transaction {transaction.id} as 'ready'"
+        )
+
+        transaction.submission_status = Transaction.SUBMISSION_STATUS.ready
+        transaction.queued_for_submit = datetime.datetime.now(datetime.timezone.utc)
+        transaction.queue = SUBMIT_TRX_QUEUE
+        await sync_to_async(transaction.save)()
+        return True
+
+    @classmethod
+    async def check_unblocked_transactions_task(cls, qa: QueueAdapter, interval: int):
+        """
+        Get the transactions that are in a 'unblocked' submission_status and
+        submit them to the SUBMIT_TRX_QUEUE for the submit_transactions_task to process.
+        The 'unblocked' submission_status implies that Polaris preivously saved the 
+        transaction as 'blocked' due to a TransactionSubmissionBlocked exception being 
+        raised by a function that submits transactions to the Stellar Network. 
+        Anchors could manually resolve an issue causing the transaction to enter 
+        the 'blocked' status and update the transaction to be "unblocked", which would allow
+        Polaris to detect and resubmit it.
+        """
+        while True:
+            await cls.process_unblocked_transactions(qa)
+            await asyncio.sleep(interval)
+
+    @classmethod
+    async def process_unblocked_transactions(cls, qa: QueueAdapter):
+        unblocked_transactions = await sync_to_async(cls.get_unblocked_transactions)()
+
+        for transaction in unblocked_transactions:
+            logger.info(
+                f"check_unblocked_transactions_task - saving transaction {transaction.id} as 'ready'"
+            )
+            transaction.submission_status = Transaction.SUBMISSION_STATUS.ready
+            transaction.queued_for_submit = datetime.datetime.now(datetime.timezone.utc)
+            transaction.queue = SUBMIT_TRX_QUEUE
+            qa.queue_transaction(
+                "check_unblocked_transactions_task", SUBMIT_TRX_QUEUE, transaction
+            )
+            await sync_to_async(transaction.save)()
```

Comment:
❓ usually I see you save the transaction before queuing, which I think is a good idea. Shouldn't we do that here?